jobs/haproxy/templates/haproxy.config.erb

```diff
@@ -169,6 +169,9 @@ end
 # }}}
 
 # Error checking
+  if !p("ha_proxy.accept_proxy", true) && p("ha_proxy.expect_proxy", true)
```

Comment:
`expect_proxy` should be the list, not a boolean.

Reply:
done